.github/workflows/test.yml

```diff
@@ -73,6 +76,27 @@ jobs:
           - ubuntu_version: "24.04"
             python_version: "3.10"
             repo_type: venv
+        exclude:
+          # Run all tests for both Docker and Podman is expensive.
```

Comment:
Since we are excluding most cases, I think this can be replaced by a single (or few) entry under `include` for the podman case, instead of excluding most things.